tui.py

```diff
@@ -0,0 +1,165 @@
+import textwrap
+import curses
+import logging
+
+
+def get_user_input(question_text):
+    screen = curses.initscr()
+    curses.echo()
+    screen.addstr(0, 0, question_text)
+    screen.refresh()
+    user_input = screen.getstr(1, 0, 50).decode('utf-8')
+    curses.noecho()
+    return user_input
+
+
+def confirm(title, question_text):
```

Comment:
I got this confirmation dialog after choosing the AppImage in my Downloads folder. I hit [Enter] to see if 'Y' was the default answer, which is usually what's implied when it's capitalized `[Y/n]`, but then I got the error `Error running TUI control panel: [Errno 2] No such file or directory: 'wine-devel_8.19-x86_64.AppImage'`.

Because of the error, the script exited with the terminal wordwrap being screwed up and the cursor being invisible. So can you add in the curses wrapper function here?

But the original error needs to be handled, too. Is it a matter of the file path only being the name and not the full path?

Reply:
I've added wrapper lines to each function.

Capitalizing just one also implies case insensitive. On some bash prompts, `yes` will be defaulted, but usually the default will be given in parentheses `(default=Y)`. Here I use `[Y/n]` to mean case insensitive.

Now, as regards the file name, I'm guessing you mean you are testing `utils.set_appimage_symlink()`. Line 1051 (only after I push the changes?) should be the only line that would call the file name:

```
appimage_filepath = f"{config.APPDIR_BINDIR}/{appimage_filename}" 
```

That would suggest that `config.APPDIR_BINDIR` isn't being added to the file path.

Reply:
I did at one time have a function for requiring an absolute path, but I didn't think it was necessary because the function has `check_appimage` which determines whether the appimage exists at all.

Reply:
Although, now that I think about it, the issue may lie here:

```
    if confirm is True or confirm == 'yes':
        logging.info(f"Copying AppImage to {config.APPDIR_BINDIR}.")
        shutil.copy(selected_appimage, f"{config.APPDIR_BINDIR}")
        os.symlink(appimage_filename, appimage_symlink_path)
        config.SELECTED_APPIMAGE_FILENAME = appimage_filename
```

`appimage_filename` in the symlink command should probably be `{config.APPDIR_BINDIR}/{appimage_filename}`. I think I'm wrongly assuming that I have switched my working directory there to the `config.APPDIR_BINDIR`.

Reply:
In committed changes right now, I did make one modification to the default var as part of this, which uses config.RECOMMENDED_WINE64_APPIMAGE_FULL_FILENAME instead of config.RECOMMENDED_WINE64_APPIMAGE_FILENAME, the only difference being whether it has the extension or not.

Reply:
I just pushed some unsquashed changes in a third commit attempting to resolve this issue.